Commons/bukkit/src/main/java/cl/bgmp/commons/modules/JoinQuitMessageModule.java

```diff
@@ -62,8 +63,24 @@ public void onPlayerQuit(final PlayerQuitEvent event) {
   @Override
   public void load() {
     if (enabled) Commons.get().registerEvents(this);
+    else Commons.get().registerEvents(new JoinQuitMessagesSuppressor());
   }
 
   @Override
   public void unload() {}
+
+  // Suppresses join & quit messages. A quick workaround to allow Commons-Bungee to handle them
+  // instead
+  private class JoinQuitMessagesSuppressor implements Listener {
+
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onPlayerJoin(final PlayerJoinEvent event) {
+      event.setJoinMessage(null);
+    }
+
+    @EventHandler
```

Comment:
What is the problem?

Reply:
Only noted because i saw the method above uses it, unsure if it's really necessary(as long as you never add more methods editing the message at a lower priority level).